funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/task_queue_publisher.py

```diff
@@ -14,37 +16,41 @@ class TaskQueuePublisher:
 
     def __init__(
         self,
-        endpoint_uuid: str,
-        pika_conn_params: pika.connection.Parameters,
+        *,
+        endpoint_id: str,
+        conn_params: pika.connection.Parameters,
```

Comment:
docstring?

Reply:
I did this one as well during the rebase. I think the others will want/need a little follow-up commit because I decided to stop making changes while rebasing.